### ISSUE TITLE
Problem decreasing required population using Docklands to import goods

### ISSUE THREAD

**Issue by ttoksie** [NiHoel/Anno1800Calculator#58]
hey mate, I've encountered a problem when enabling docklands to import goods, so far I have only had it happen to engineer's, no other class.

Say I set out with how many investors I want on my islands, that tells me how many engineer's I will need, so I input that into how many engineer's I have to figure out production chains, If I then decide to import a good that engineer's produce (say light bulbs in exchange for cigars at x2.0) the calculator will update with how many extra cigar productions chains I need, and also tell me I need fewer engineer's, however when I input that lower number into the calculator the number of engineer's it output's go's up drastically.

so far the workaround I have found is that I can only increase my number of engineers, in order to get an accurate number required, so start the calculator with only the investors slot filled in, and input a low number (say 100) into every other resident type, and go down the list to disallow consumption of all of the good I plan on importing, set up the docklands trade feature, to see how many engineer's I will need to produce what I'm not importing, and then go back down to check the allow consumption of those goods which will have the calculator suggest I need more engineers, then go back to all of the good i'm importing and update the amounts imported again.

If you like I can take some screenshot of the issue and hopefully that will help me explain it better if my explanation here is hard to follow.

Thanks for all of the work you've put into this calculator, it has revolutionised the game for me.

**Comment by NiHoel** [comment]
Please send me the configuration (see download button) before and after this increase happens.

**Comment by ttoksie** [comment]
Hey mate, I actually figured the problem out, and it is now working correctly.

My browser was auto-refreshing the page every time I made a change, and in doing so was resetting the origin of goods, I did my initial setup with wood veneers made in the new world, and when my browser would auto refresh it would revert that setting back to "same region as consumer" which meant it was calculating for engineers to make veneers and thus greatly increasing the number of engineers I would need.

I'm sorry for wasting your time with the false bug report when it ended up being a problem on my end.

Thanks again for creating this calculator, it's a great tool and really makes the game much more enjoyable for me.

________________________________
